src/Server/Main.idr

```diff
@@ -118,7 +118,7 @@ main = do
               addLibDir cwd
               o <- newRef ROpts (Opts.defaultOpts Nothing (REPL False) [])
               u <- newRef UST initUState
-              m <- newRef MD initMetadata
+              m <- newRef MD (initMetadata "")
```

Comment:
I think the convention is "(interactive)", but these aren't well documented, so I could be wrong.